Scheduler/observing_blocks_maintel/BLOCK-T74/BLOCK-T74_1.json

```diff
@@ -0,0 +1,412 @@
+{
+    "name": "BLOCK-T74_1",
+    "id": "e7afd17e-a473-4890-a2e0-510582b318ad",
+    "program": "BLOCK-T74_1",
```

Comment:
this needs to change to be compliant with the block naming convention: `BLOCK-T74`.

Reply:
Is this convention written down somewhere?  Do we also have a convention for `"reason"` ?